OpenRA.Game/Support/Benchmark.cs

```diff
@@ -50,7 +51,7 @@ public void Write()
 				Log.Write(name, "tick,time [ms]");
 
 				foreach (var point in sample.Value)
-					Log.Write(name, $"{point.Tick},{point.Value}");
+					Log.Write(name, $"{point.Tick},{point.Value.ToString(CultureInfo.InvariantCulture)}");
```

Comment:
perhaps add a comment explaining this

Reply:
No, we use invariant culture everywhere. That it was removed or not added here is just a bug.